lms/djangoapps/ccx/views.py

```diff
@@ -45,6 +45,7 @@
     get_email_params,
 )
 
+from labster_course_license.licensed_blocks_override import is_visible_to_staff_only
```

Comment:
Can you please import custom entities in the end of import list?
Try to follow this order
- python libs (requests, datetime, re, os etc.)
- django/django rest libs (views, shortcuts, utils etc.)
- custom libs (labster_course_license etc.)

Reply:
Changed, Thank you :+1: